tests/unit-tests/scene/test_surface_stack.cpp

```diff
@@ -140,6 +140,7 @@ struct SurfaceStack : public ::testing::Test
     std::shared_ptr<ms::SceneReport> const report = mr::null_scene_report();
     ms::SurfaceStack stack{report};
     void const* compositor_id{&default_params};
+    geom::Rectangle view_area;
```

Comment:
Is this some left-over code? I don't see what it connects to.

Reply:
Jup, left over from the first attempt that required the DisplayBuffers view_are to be passed in to `scene_elements_for`